docs/mast/mast.rst

```diff
@@ -1120,6 +1120,62 @@ To list the available deep field surveys at a particular location there is `~ast
    ['candels_gn_60mas', 'candels_gn_30mas', 'goods_north']
 
 
+HAPCut
```

Comment:
side note: the doctest are failing in an earlier example, so these has not been run during my testing